### DISCUSSION TITLE
b1更新至b2

### DISCUSSION THREAD

**Discussion by AquamarineCyan** [nonebot/discussions#142] [问答]
使用`pip install nonebot2 -U`更新，但是本地文件夹仍为`nonebot_adapter_cqhttp-2.0.0b1.dist-info`，文件修改日期并未发生变化

是否需要其他更新命令

**Comment by MingxuanGame** [comment]
`adapter-onebot` (`adapter-cqhttp`)目前并未发布 beta2，无需更新

**Reply by AquamarineCyan** [reply]
如果以后发布，是否会提示，或者说在nb2的部署文档中考虑加入命令提示

**Reply by MingxuanGame** [reply]
> 或者说在nb2的部署文档中考虑加入命令提示

命令提示不应写入文档，因为不同的包管理器的更新方式不同

**Reply by AquamarineCyan** [reply]
那就是直接更新适配器即可
